### PR TITLE
libvirt 1.3.5

### PR DESCRIPTION
**Pull request by nijikon**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/libvirt.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Libvirt < Formula
   desc "C virtualization API"
   homepage "https://www.libvirt.org"
-  url "https://libvirt.org/sources/libvirt-1.3.4.tar.gz"
-  sha256 "e2396ebebb3f3fdb50429ce8faa99559f6e8e3cc0493d5fa0c1999db189c25bd"
+  url "https://libvirt.org/sources/libvirt-1.3.5.tar.gz"
+  sha256 "93a23c44eb431da46c9458f95a66e29c9b98e37515d44b6be09e75b35ec94ac8"
+  head "https://github.com/libvirt/libvirt.git"
```
Comment (DomT4) [inline Formula/libvirt.rb]: Upstream doesn't seem to keep a pre-generated `configure` script around so this won't actually work as-is.

Comment (nijikon) [inline Formula/libvirt.rb]: Oh, i've missed that, thanks for keeping an eye out ;)

**Comment by nijikon** [comment]
#1568

**Comment by nijikon** [comment]
@DomT4 head removed.

**Comment by UniqMartin** [comment]
:+1:

**Comment by DomT4** [comment]
👍

**Pull request closed without merging**